netket/vqs/base.py

```diff
@@ -304,10 +305,10 @@ def to_qobj(self):  # -> "qutip.Qobj"
         Returns:
             A :class:`qutip.Qobj` object.
         """
-        from qutip import Qobj
+        qutip = import_optional_dependency("qutip", descr="to_qobj")
```

Comment:
looks like there is several unrelated commits

Reply:
That's me adding it. The full summation code uses qutip (otherwise we'd have to implement a partial trace by hand) and I wanted a clean error message. I can get it this commit in a separate PR if you prefer.

Reply:
it's fine